video-presentation.py

```diff
@@ -246,13 +285,19 @@ def find_slides_transitions(self, connector, host, secret_key, resource, masks=N
             self.logger.debug("Using advanced algorithm for finding slides. settings: %s", settings)
             results = self.slide_find_advanced(resource['local_paths'][0], masks=masks, **settings)
 
+        # Create and upload the compressed previews
+        mp4_preview, webm_preview = self.create_video_previews(resource['local_paths'][0])
+        mp4_preview_id = pyclowder.files.upload_preview(connector, host, secret_key, resource['id'], mp4_preview, {})
```

Comment:
Not sure if I should add something in the metadata arg

Reply:
I also didn't use it.